### PR TITLE
Merge xUnit.net 2.0-rc3 changes

### PR DESCRIPTION
**Pull request by bricelam**
The only thing to review here is 44a5d250853a015585d2ee8af018231d5e21ae8e. Everything else is from upstream.

Note, the package name has been updated to `xunit.runner.aspnet`. All of our repos will need to react with a `project.json` like the following.

```JSON
{
    "dependencies": {
        "xunit.runner.aspnet": "2.0.0-aspnet-*"
     },
     "commands": {
         "test": "xunit.runner.aspnet"
    }
}
```

### CODE REVIEW

**Comment by bricelam** [comment]
I would like to dedicate this PR to @dougbu

File: src/xunit.runner.aspnet/Program.cs [hunk 1]
```diff
@@ -249,28 +299,48 @@ XElement ExecuteAssembly(object consoleLock, string defaultDirectory, XunitProje
...
                     discoveryVisitor.Finished.WaitOne();
 
-                    IDictionary<ITestCase, VsTestCase> vsTestcases = null;
-                    if (designTime)
-                    {
-                        vsTestcases = DesignTimeTestConverter.Convert(discoveryVisitor.TestCases);
-                    }
+                    //IDictionary<ITestCase, VsTestCase> vsTestcases = null;
+                    //if (designTime)
+                    //    vsTestcases = DesignTimeTestConverter.Convert(discoveryVisitor.TestCases);
```
Comment (dougbu) [inline src/xunit.runner.aspnet/Program.cs]: is this commenting in the upstream repo?  if not suggest just deleting the code

File: src/xunit.runner.aspnet/Program.cs [hunk 2]
```diff
@@ -308,26 +375,27 @@ XElement ExecuteAssembly(object consoleLock, string defaultDirectory, XunitProje
...
-                                             select t.Key)
-                                            .ToList();
-                    }
+                    //else
+                    //{
+                    //    filteredTestCases = (from t in vsTestcases
+                    //                         where designTimeFullyQualifiedNames.Contains(t.Value.FullyQualifiedName)
+                    //                         select t.Key)
+                    //                        .ToList();
+                    //}
```
Comment (dougbu) [inline src/xunit.runner.aspnet/Program.cs]: again, if this is a local change, just delete

File: tools/kvm.ps1 [hunk 3]
```diff
@@ -0,0 +1,824 @@
+param(
```
Comment (dougbu) [inline tools/kvm.ps1]: why is this file checked into the repo?

**Comment by dougbu** [comment]
is the change to `xunit.runner.aspnet` coming from the upstream repo?  not sure the name is right for a general .NET 5.0 testing framework.

**Comment by dougbu** [comment]
my comments are all outside the 44a5d25 commit.  so :shipit: if that's all we can change.

**Comment by bricelam** [comment]
This depends on aspnet/xunit#1

**Pull request merged**